navpatch/repositories/fs.go

```diff
@@ -0,0 +1,75 @@
+package repositories
```

Comment:
I'd rather keep a flat package structure here unless there's a good reason not to.